jpasskit/src/main/java/de/brendamour/jpasskit/PKNFC.java

```diff
@@ -22,9 +22,6 @@
     private static final long serialVersionUID = -2017873167088954297L;
     private String message;
     private String encryptionPublicKey;
-    
-    public PKNFC() {
```

Comment:
As I know, default constructor is necessary for correct deserialization in case of using "native" Java serialization mechanism.

Reply:
Yeah. I'm not too sure about this either. It should be fine, but it won't hurt

Reply:
Maybe the issue happens only if you override the default constructor, so it may not be the case here.